.circleci/config.yml

```diff
@@ -0,0 +1,18 @@
+version: 2.1
+orbs:
+  node: circleci/node@1.1
+jobs:
```

Comment:
do you know if this will this run multiple jobs or just one?

Reply:
i'm just gonna approve it for now, and if it doesn't we can add another version of node or something to trigger another job